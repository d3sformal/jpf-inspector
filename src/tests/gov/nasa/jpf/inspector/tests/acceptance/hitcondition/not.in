cr bp it = any and b == 9 and not a == 5
run
wait
thpc
del bp 1