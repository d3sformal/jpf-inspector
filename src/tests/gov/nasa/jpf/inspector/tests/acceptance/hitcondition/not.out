INFO: JPF created and connected, SuT is started
INFO: Breakpoint hit: [1 :  state=ENABLED hits=1 hitsTotal=1 ( instruction_type=any ) and ( ( (#thread.#stackFrame.b == 9) ) and ( not (#thread.#stackFrame.a == 5) ) )]
	SuT (Thread=0) executes the gov/nasa/jpf/inspector/tests/acceptance/sut/Singlepath.java:8 - return  gov.nasa.jpf.inspector.tests.acceptance.sut.Singlepath.main([Ljava/lang/String;)V
0 : gov/nasa/jpf/inspector/tests/acceptance/sut/Singlepath.java:8:  }
	gov.nasa.jpf.inspector.tests.acceptance.sut.Singlepath:main:12:return  gov.nasa.jpf.inspector.tests.acceptance.sut.Singlepath.main([Ljava/lang/String;)V
Breakpoint with ID 1 successfully deleted.
INFO: JPF Terminating